src/Commands/NotificationCommand.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types=1);
+
+namespace Soyhuce\Somake\Commands;
+
+use Illuminate\Console\Command;
+use Soyhuce\Somake\Commands\Concerns\AsksDomain;
+use Soyhuce\Somake\Support\Finder;
+use Soyhuce\Somake\Support\Writer;
+use function Laravel\Prompts\outro;
+use function Laravel\Prompts\text;
+
+class NotificationCommand extends Command
+{
+    use AsksDomain;
+
+    /** @var string */
+    public $signature = 'somake:notification';
+
+    /** @var string */
+    public $description = 'Generates a Notification in Domain';
+
+    public function handle(Finder $finder, Writer $writer): void
+    {
+        $notification = text(label: 'What is the Notification name ?', required: true);
+
+        $queued = $this->confirm(question: 'Should the notification be queued ?');
+
+        $multilingual = $this->confirm(question: 'Should the notification be multilingual ?', default: true);
```

Comment:
not sure about the fact that translations should be an option. IMHO it's better to enable this by default

Reply:
Done in b833bbbd091cd261c022bef4beb88cd06e9b9483